src/services/buildService.ts

```diff
@@ -24,6 +24,16 @@ class BuildService {
   public getBuildCount(requestConfig: AxiosRequestConfig = {}) {
     return pncClient.getHttpClient().get(`${this.path}/count`, requestConfig);
   }
+
+  /**
+   * Gets dependency graph for a build.
+   * @returns DependencyGraph
+   */
+  public getDependencyGraph(id: string, requestConfig: AxiosRequestConfig = {}) {
+    if (id) {
```

Comment:
if `id` is falsy, it's developer mistake? if yes, some error should be fired?

Reply:
![image](https://user-images.githubusercontent.com/6870455/185397372-ce099a9c-9ff4-4525-9708-8498006c2697.png)